use_case_guide/multi_tenant.rst

```diff
@@ -0,0 +1,517 @@
+
+.. image:: ../images/zipf.png
+
+For instance, in a database of 100 tenants, the largest is predicted to account for 23% of the data. In a more realistic example for a large SaaS company, if there are 10k tenants, the largest will account for ~2% of the data. Even at 10TB of data, the largest tenant will require 200GB, which can pretty easily fit on a single node.
+
+Another question is regarding performance when large and small tenants are on the same node. Standard shard rebalancing will improve overall performance but it may or may not improve the mixing of large and small tenants. The rebalancer simply distributes shards to equalize storage usage on nodes, without examining which tenants are allocated on each shard.
+
+To improve resource allocation and make guarantees of tenant QoS it is worthwhile to move large tenants to dedicated nodes. Citus provides the tools to do this.
+
+In our case, let's imagine that our old friend company id=5 is very large. We can isolating the data for this tenant happens in two somewhat low-level steps. We'll present the commands here, and you can consult :ref:`tenant_isolation` for the theory behind them.
+
+First consolidate the tenant's data into a bundle (called a shard) suitable to move. The CASCADE option also applies this change to the rest of our tables distributed by :code:`company_id`.
+
+::
+
+
+  SELECT isolate_tenant_to_new_shard(
+    'companies', 5, 'CASCADE'
+  );
+
+The output is the shard id dedicated to hold :code:`company_id=5`:
+
+.. code-block:: text
+
+  ┌─────────────────────────────┐
+  │ isolate_tenant_to_new_shard │
+  ├─────────────────────────────┤
+  │                      102240 │
+  └─────────────────────────────┘
+
+Next we move the data across the network to a new dedicated node. Create a new node as described in the previous section. Take note of its hostname as shown in the Nodes tab of the Cloud Console.
+
+::
+
+  -- find the node currently holding the new shard
+
+  SELECT nodename, nodeport
+    FROM pg_dist_shard_placement
+   WHERE shardid = 102240;
+
+  -- move the shard to your choice of worker (it will also move the
+  -- other shards created with the CASCADE option)
+
+  SELECT master_move_shard_placement(
+    102240,
+    'source_host', source_port,
+    'dest_host', dest_port);
+
+You can confirm the shard movement by querying :ref:`pg_dist_shard_placement <placements>` again.
+
+Where to Go From Here
+---------------------
+
+Now that you know the basics of designing a schema for scalability in a multi-tenant application, you can continue further in our docs to put this knowledge to use. If you have an existing schema and want to migrate it for Citus, see :ref:`Multi-Tenant Transitioning <transitioning_mt>`. To adjust a front-end application, specifically Ruby on Rails, read :ref:`rails_migration`. Finally, try :ref:`Citus Cloud <cloud_overview>`, the easiest way to manage a Citus cluster, available with discounted developer plan pricing.
```

Comment:
With this, you now know how to use Citus to power your multi-tenant application for scalability. If you have an existing application and want to migrate it to Citus, see MT transitioning. To ...